app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command = command.split()
+    copy_command = command[0]
+    source_file = command[1]
+    target_file = command[2]
+    if len(command) == 3:
```

Comment:
Need to check before setting variables

Reply:
if len(command) == 3:

Should be checked before 

   ` copy_command, source_file, target_file = command.split()`